blockly-rtc/typedefs.js

```diff
@@ -23,17 +23,15 @@
 /**
  * An entry from the database.
  * @typedef {Object} Entry
- * @property {<!Array.<!Object>>} events An array of Blockly Events in JSON
- * format.
+ * @property {<!Array.<!Blockly.Events>>} events An array of Blockly Events.
```

Comment:
``<!Array.<!Blockly.Event>>`` not Blockly.Events

Reply:
Done.